fusillade/api/groups/__init__.py

```diff
@@ -13,8 +15,12 @@ def put_new_group(token_info: dict):
     return make_response(f"New role {json_body['group_id']} created.", 201)
 
 
-def get_groups():
-    pass
+def get_groups(token_info: dict):
+    assert_authorized(token_info['https://auth.data.humancellatlas.org/email'],
```

Comment:
Would it make sense to make this `assert_authorized` function into a decorator?

Reply:
i'll do that in a different PR